docs/Configuration/fleet-server-configuration.md

```diff
@@ -2213,9 +2213,9 @@ Name of the S3 bucket for storing software and file carves.
     bucket: some-bucket
   ```
 
-##### s3_prefix
+##### s3_software_installers_prefix
 
-Prefix to prepend to software and file carves.
+Prefix to prepend to software.
 
 All carve objects will also be prefixed by date and hour (UTC), making the resulting keys look like: `<prefix><year>/<month>/<day>/<hour>/<carve-name>`.
```

Comment:
Since this is now the section for `software installers`, this sentence should be moved to the one for carves, it does not apply for installers.

Reply:
Thanks for catching that!